diff --git a/presage-cli/src/main.rs b/presage-cli/src/main.rs
index 8ff5d38..3f7e05c 100644
--- a/presage-cli/src/main.rs
+++ b/presage-cli/src/main.rs
@@ -169,7 +169,7 @@ enum Cmd {
         group(
             ArgGroup::new("list-messages")
                 .required(true)
-                .args(&["recipient_uuid", "group_master_key"])
+                .args(&["recipient_service_id", "group_master_key"])
         )
     )]
     ListMessages {
